src/runtime_src/core/edge/drm/zocl/zocl_aie.c

```diff
@@ -99,6 +221,19 @@ zocl_create_aie(struct drm_zocl_dev *zdev, struct axlf *axlf)
 			DRM_ERROR("Fail to allocate memory.\n");
 			goto done;
 		}
+
+		zdev->aie->err.errors = vzalloc(sizeof (struct aie_error) *
+		    ZOCL_AIE_ERROR_CACHE_CAP);
```

Comment:
Why the word wrap?  Would it be easier to read if the code as all on one line.

Reply:
Linux kernel code has a coding style that requires breaking long lines (maximum 80 characters per line). checkpatch tool will also enforce it.

https://www.kernel.org/doc/html/v4.10/process/coding-style.html

Reply:
Thank you for the reminder.